### PR TITLE
 BUMP BMO to latest patch versions v0.5.1 and v0.4.2 

### PR DESCRIPTION
**Pull request by kashifest**
as the title says.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ The following table describes which branches are tested for different test trigg
 | test suffix | CAPM3 branch | IPAM branch  | BMO branch/tag  | Keepalived tag | Ironic tag |
 | ----------- | ------------ | -----------  | --------------- | -------------- | ---------- |
 | main        | main         | main         | main            | latest         | latest     |
-| release-1-6 | release-1.6  | release-1.6  | release-0.5     | v0.5.0         | v24.0.0    |
+| release-1-6 | release-1.6  | release-1.6  | release-0.5     | v0.5.1         | v24.0.0    |
 | release-1-5 | release-1.5  | release-1.5  | release-0.4     | v0.4.1         | v23.1.0    |
```
[the remark this answers is not in the document]
Comment (kashifest) [reply README.md]: Done

**Review by tuminoid — APPROVED** [review]

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Review by mboukhalfa — APPROVED** [review]

**Pull request merged**